BookStore/src/main/java/ro/utcn/fulgadan/BookStore/business/CartService.java

```diff
@@ -0,0 +1,36 @@
+package ro.utcn.fulgadan.BookStore.business;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ro.utcn.fulgadan.BookStore.data.entity.Product;
+import ro.utcn.fulgadan.BookStore.data.entity.ShoppingCart;
+import ro.utcn.fulgadan.BookStore.data.entity.User;
+import ro.utcn.fulgadan.BookStore.data.repository.CartJpaRepository;
+
+import java.util.List;
+
+@Service
+public class CartService {
+    @Autowired
+    CartJpaRepository cartJpaRepository;
```

Comment:
I generally recommend that you use constructor injection instead of field injection (as we did in the lab): https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/